app/views/api/v1/targets/index.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+json.targets @targets.each do |target|
```

Comment:
It looks like you should add this file in the PR where you added the code related to the index logic, not in this PR